varnish/tests/test_unit.py

```diff
@@ -0,0 +1,207 @@
+# (C) Datadog, Inc. 2018
```

Comment:
It might be cool to use markers to distinguish between unit and integration tests, so part of the tests can be run without needing a Docker daemon running, see `redisdb` for an example

Reply:
I did use a marker to mark integrations tests. Is it better to also specify one for unit tests ? Or is `pytest  -m"not integration"` good enough ?

Reply:
`pytest -m"not integration"` is perfect, let's add it to tox